packages/embark-ui/src/components/ContractOverview.js

```diff
@@ -196,13 +209,22 @@ class ContractFunction extends Component {
          <Collapse isOpen={this.state.functionCollapse} className="relative">
           <CardBody>
             {ContractFunction.isPayable(this.props.method) &&
-              <Form inline>
-                <Label for={this.props.method.name + '-value'} className="mr-2 font-weight-bold contract-function-input">ETH</Label>
+              <Form inline className="mb-1">
+                <Label for={this.props.method.name + '-value'} className="mr-2 font-weight-bold contract-function-input">Transaction value</Label>
                 <Input name={this.props.method.name}
                        id={this.props.method.name + '-value'}
                        onChange={(e) => this.handleChange(e, this.props.method.name + '-value')}
                        onKeyPress={(e) => this.handleKeyPress(e)}
+                       invalid={this.state.unitConvertError && this.state.unitConvertDirty ? true : null}
+                       valid={!this.state.unitConvertError && this.state.unitConvertDirty ? true : null}
```

Comment:
I might be missing something, but shouldn't it be enough to go with either `valid` **or** `invalid`? One implies the other, no?

Reply:
Yes, but it could be me who is missing something. I could not figure out a way to define attributes conditionally, and instead resorting to making them `null` when unneeded. Is there a better way to achieve this?

Reply:
I think `valid` puts a green border or something maybe when it's valid? Depends if we want that.
As for using `null`, I think just doing ` valid={!this.state.unitConvertError && this.state.unitConvertDirty}` should work no?